lib/common/formats.js

```diff
@@ -38,20 +38,20 @@ function fileHeader(options, commentStyle) {
   return to_ret;
 }
 
-function variablesWithPrefix(prefix, properties, commentStyle) {
+function variablesWithPrefix(prefix, properties, commentStyle, separator = ':') {
   return properties.map(function(prop) {
-      var to_ret_prop = prefix + prop.name + ': ' + (prop.attributes.category==='asset' ? '"'+prop.value+'"' : prop.value) + ';';
+    var to_ret_prop = prefix + prop.name + separator + ' ' + (prop.attributes.category === 'asset' ? '"' + prop.value + '"' : prop.value) + ';';
 
-      if (prop.comment) {
-        if (commentStyle === 'short') {
-          to_ret_prop = to_ret_prop.concat(' // ' + prop.comment);
-        } else {
-          to_ret_prop = to_ret_prop.concat(' /* ' + prop.comment + ' */');
-        }
+    if (prop.comment) {
+      if (commentStyle === 'short') {
+        to_ret_prop = to_ret_prop.concat(' // ' + prop.comment);
+      } else {
+        to_ret_prop = to_ret_prop.concat(' /* ' + prop.comment + ' */');
```

Comment:
I've moved the indentation alignment in a separate commit - if you feel like we should move this out of this PR I'll happily do this.

Reply:
Looks good to me! Cleaning up indentation and alignment is always welcome. We should probably use prettier or something in the future though... That'll be a different PR

Reply:
Sounds like a good idea! I've removed the indentation cleanup for now.